docs/dockerfile-frontend-options.md

```diff
@@ -0,0 +1,65 @@
+# Dockerfile Frontend Build Options
+
+## use with `buildctl build --opt`
+For `dockerfile.v0` frontend there are some parameters that can be used with `buildctl build --opt`.
+The following parameters are the most frequently used.
+* target: set the target build stage to build.
+  * Example: `target=Stage1`
+* filename: name of the Dockerfile (Default is 'Dockerfile'),set to use a differently-named Dockerfile.
+  * Example: `filename=dockerfile.prod`,`filename=other.dockerfile`
+* dockerfilekey: name of the custom named context for a directory which includes Dockerfile. Useful when there is more than one build source.
```

Comment:
This is advanced. If you want you could explain the `--local` values that Dockerfile takes. If you do that then `dockerfilekey` and `contextkey` can be used to overwrite local names. But the reader first needs to understand how `--local` works.